### ISSUE TITLE
Constants are required when using FromUri

### ISSUE THREAD

**Issue by spencerkittleson** [LambdaSharp/LambdaSharpTool#130]
When using [FromUri] in a `ALambdaApiGatewayFunction` constants are required parameter by API Gateway. In this example, `TEST` is required. It shouldn't be.

Module.yml
```yml
 - Function: APILambda
    Description: Lambda function to handle API request/responses
    Memory: 512
    Timeout: 30
    Sources:
      - Api: GET:/
        Invoke: GetItems
```

```c# 
MyResponse GetItems([FromUri] RequestModel) { ... }

public class RequestModel {

        //--- Constants ---
        public const string TEST = "foo";

        //--- Properties ---
        [JsonProperty("filter")]
        public string Filter {
            get; set;
        }
}
```

**Comment by bjorg** [comment]
How is `TEST` required?

**Comment by bjorg** [comment]
In this sample, there is no `const` and it seems to be doing the exact same thing you want to do above. What am I missing?
https://github.com/LambdaSharp/LambdaSharpTool/blob/master/Samples/ApiInvokeAssemblySample/MyFunction/Logic.cs#L75

**Comment by spencerkittleson** [comment]
Requiring a `const` as a input parameter for an API Gateway doesn't serve a purpose.  It doesnt get fill in. API Gateway will reject the request

**Comment by bjorg** [comment]
Fixed in 0.7.0.8.
